rules/alert-container-optimized-os-not-in-use/raw.rego

```diff
@@ -0,0 +1,34 @@
+package armo_builtins
+import future.keywords.in
+
+
+# checks if a node is not using a "Container-Optimized OS". 
+# "Container-Optimized OS" prefixes are configured in 'container_optimized_os_prefixes'.  
+# deny if 'nodes.status.nodeInfo.osImage' not starting with at least one item in 'container_optimized_os_prefixes'.
+deny[msga] {
+
+	nodes := input[_]
+	nodes.kind == "Node"
+
+	# list of "Container-Optimized OS" images prefixes 
+	container_optimized_os_prefixes = ["Bottlerocket"]
```

Comment:
There's only one known Container-Optimized OS?

Reply:
For EKS, this is the most common that is also the one CIS mentioned in the control, so I went on with it for now.